<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchwindow</class>
 <widget class="QWidget" name="searchwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>503</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    border-image:url(:/back/5.jpg);
    border: 1px solid #32435E;
    background:rgb(255, 255, 0);
    border-radius: 5px;
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>98</x>
     <y>20</y>
     <width>541</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_1">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">color:black;
font: 12pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>搜索：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_line"/>
      </item>
      <item>
       <widget class="QPushButton" name="search_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>楷体</family>
          <pointsize>12</pointsize>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label2">
        <property name="styleSheet">
         <string notr="true">color:black;
font: 12pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>成长值设置：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="growth_line"/>
      </item>
      <item>
       <widget class="QLabel" name="growth_value_label">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="growth_value_button">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="name_label">
        <property name="styleSheet">
         <string notr="true">color:black;
font: 12pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>名字：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="personality_label">
        <property name="styleSheet">
         <string notr="true">color:black;
font: 12pt &quot;楷体&quot;;</string>
        </property>
        <property name="text">
         <string>性格：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="status_label">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;楷体&quot;;
color:black;</string>
      </property>
      <property name="text">
       <string>身份：</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QTextEdit" name="introduction">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;楷体&quot;;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>460</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;
background-color:rgbr(255,255,0,0);</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
